ConsoleApp/School/ClassRoom.cs

```diff
@@ -0,0 +1,16 @@
+namespace ConsoleApp.School;
+public enum ClassRoom
+{
+    _101,
```

Comment:
as we've discussed during the lesson enum is not the best for storing rooms number. it's better to store it in int or string